metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlGetPartition.java

```diff
@@ -115,7 +123,6 @@ public int getPartitionCount(
         final QualifiedName tableName
     ) {
         final long start = registry.clock().wallTime();
-        final Integer result;
```

Comment:
Probably my style of coding. I like to define a result at the beginning of the method, assign a value to it during the method and have one return statement at the end of the method.